jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -645,6 +648,25 @@ EOF
   [ "${result}" = "${expected}" ]
 }
 
+@test "Verify if the jbpm required modules are contained in the jboss-deployment-structure.xml when KIE_SERVER_JBPM_CLUSTER is true" {
+  export KIE_SERVER_JBPM_CLUSTER="true"
+  configure_jbpm_cache_modules
+  #this is the return of xmllint --xpath "//*[local-name()='module'][@name='org.infinispan']/@services" /opt/eap/standalone/deployments/ROOT.war/WEB-INF/jboss-deployment-structure.xml
+  expectedInfinispan=' services="export"'
```

Comment:
shouldn't you validate all the added content just only this line?

Reply:
No, the content added is just "<module name="org.infinispan" services="export"/><module name="org.jgroups"/>" this line 655 check infinispan tag, the line 663 is the check for jgroups tag

Reply:
ok..